docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -47,15 +47,55 @@ handling user interactions, and managing search functionality.
 Actions are an optional way to allow users to perform custom tasks such as
 adding a new item to the list of options.
 
-#### Callback
+### Selection behavior
 
-`onSelect` is the recommended callback handler that should be used in most cases. This will immediately inform
-the consumer of changes to selections as they happen.
+#### Single-select
 
-If however immediate changes are not desired, and instead it is preferable to wait until all selections or
-modifications to selections have been made before informing the consumer, `onClose` may be used. The arguments provided to this callback are
-Identical to those of `onSelect`, only the timing differs. Please note this will only
-have a noticeable effect with `multiSelect = true` since the Combobox closes immediately upon selection with single-select.
+Single-select is ComboBox's default selection mode. When a selection is made,
+the Combobox will close.
+
+#### Multi-select
```

Comment:
Not sure if it's a bug in storybook or the combobox itself, but when I go to the Multiselect web story and open up the combobox, then I go back to the Docs page without closing the combobox, the combobox stays on screen - can we fix that?

Reply:
I think Chris was going to look for a ticket about this - since, like we talked about in standup, this is a known issue with other components as well

Reply:
Ok we can address it in a separate ticket if it's already an existing issue.

Reply:
yeah, it's because of the `ReactDOM.createPortal` that we use to solve another issue

you can see the same thing in Storybook if you open a `Modal` and then go elsewhere